ccc-js/src/main/javascript/package-res/ccc/core/cartesian/axis/cart-axis.js

```diff
@@ -964,10 +971,12 @@ pvc_CartesianAxis.options({
         resolve: '_resolveFull'
     },
 
-    TicksRegularity: {
+    PreserveTickAlignment: {
       resolve: '_resolveFull',
       cast: Boolean,
-      value: false
+      getDefault: function() {
+        return this.chart.options.slidingWindow;
```

Comment:
Simply looking at the `slidingWindow` option like this will not always work.
Please look at https://github.com/webdetails/ccc/blob/master/ccc-js/src/main/javascript/package-res/ccc/core/base/slidingWindow.js#L78 (and this one has unit tests ;-) )